Source/NSOutlineView.m

```diff
@@ -2198,10 +2203,50 @@ - (NSCell *) preparedCellAtColumn: (NSInteger)columnIndex row: (NSInteger)rowInd
 	  cell = [tb dataCellForRow: rowIndex];
 	}
     }
-  
+
   return cell;
 }
 
+- (NSView *) viewAtColumn: (NSInteger)column row: (NSInteger)row makeIfNecessary: (BOOL)flag
+{
+  NSTableColumn *tb = [_tableColumns objectAtIndex: column];
+  NSIndexPath *path = [NSIndexPath indexPathForItem: column
+					  inSection: row];
+  NSView *view = [self _renderedViewForPath: path];
+  NSRect drawingRect = [self frameOfCellAtColumn: column
+					     row: row];
+  id item = [self itemAtRow: row];
+
+  if (tb == _outlineTableColumn)
+    {
+      drawingRect = [[GSTheme theme] drawOutlineCell: tb
+					 outlineView: self
+						item: item
+					 drawingRect: drawingRect
+					    rowIndex: row];
+    }
+  
+  if (view == nil
+      && flag == YES)
+    {
+      if ([_delegate respondsToSelector: @selector(outlineView:viewForTableColumn:item:)])
+	{
+	  view = [_delegate outlineView: self
+		     viewForTableColumn: tb
+				   item: item];
+	}
+      else
+	{
+	  view = [self _prototypeCellViewFromTableColumn: tb];
+	}
+    }
+
+  [view setFrame: drawingRect];
+  [self _setRenderedView: view forPath: path];
```

Comment:
Same here, only store the view if it was newly created.

Reply:
Done.